### ISSUE TITLE
pip install jsonobject results in error message

### ISSUE THREAD

**Issue by kbrose** [dimagi/jsonobject#149]
In a fresh environment, running

```
pip install jsonobject
```

prints a bunch of error text, the main part of which appears to be the following:

```
  ...
  running install_egg_info
  running egg_info
  writing requirements to jsonobject.egg-info\requires.txt
  writing jsonobject.egg-info\PKG-INFO
  writing top-level names to jsonobject.egg-info\top_level.txt
  writing dependency_links to jsonobject.egg-info\dependency_links.txt
  reading manifest file 'jsonobject.egg-info\SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  writing manifest file 'jsonobject.egg-info\SOURCES.txt'
  Copying jsonobject.egg-info to build\bdist.win-amd64\wheel\.\jsonobject-0.9.3-py2.7.egg-info
  running install_scripts
  C:\Users\kevin.rose\AppData\Local\Continuum\Anaconda2\envs\sdk-test\lib\site-packages\wheel\pep425tags.py:75: RuntimeWarning: Config variable 'Py_DEBUG' is unset, Python ABI tag may be incorrect
    warn=(impl == 'cp')):
  C:\Users\kevin.rose\AppData\Local\Continuum\Anaconda2\envs\sdk-test\lib\site-packages\wheel\pep425tags.py:79: RuntimeWarning: Config variable 'WITH_PYMALLOC' is unset, Python ABI tag may be incorrect
    warn=(impl == 'cp')):
  C:\Users\kevin.rose\AppData\Local\Continuum\Anaconda2\envs\sdk-test\lib\site-packages\wheel\pep425tags.py:85: RuntimeWarning: Config variable 'Py_UNICODE_SIZE' is unset, Python ABI tag may be incorrect
    sys.version_info < (3, 3))) \
  error: [Errno 2] No such file or directory: 'LICENSE'

  ----------------------------------------
  Failed building wheel for jsonobject
  Running setup.py clean for jsonobject
Failed to build jsonobject
Installing collected packages: six, jsonobject
  Running setup.py install for jsonobject ... done
Successfully installed jsonobject-0.9.3 six-1.11.0
````

The installation does appear successful (`jsonobject` imports without error).

**Comment by rightparen** [comment]
I am seeing this as well.

**Comment by kbrose** [comment]
Should be closed by #153
